KomaMRICore/test/test_files/Utils.jl

```diff
@@ -0,0 +1,72 @@
+using HDF5
+
+function signal_jemris()
+    path = @__DIR__
+    sig = h5open(joinpath(path, "jemris_signals_epi_sphere_cs.h5"))["/signal/channels/00"]
+    sig = sig[1,:] + 1im*sig[2,:]
+    sig = sig[:]
+    return sig
+end
+
+function phantom_sphere()
+    path = @__DIR__
+    fid = h5open(joinpath(path, "phantom_sphere.h5"), "r")
+    x, y, z, ρ = fid["x"][:], fid["y"][:], fid["z"][:], fid["ρ"][:]
+    T1, T2, T2s, Δw = fid["T1"][:], fid["T2"][:], fid["T2s"][:], fid["Δw"][:]
+    return Phantom(; name="sphere", x, y, z, ρ, T1, T2, T2s, Δw)
+end
+
+function seq_epi_100x100_TE100_FOV230()
+
+    # Gyromagnetic constant
+    γ_jemris = 2π*γ*1e-9            # γ in [Hz/T]. γ_jemris in [rad/us/mT]
+
+    # For excitation
+    Trf = 100e-6                   # RF duration 100us
+    Arf = (π/2)/(2π*γ*Trf)         # RF Amplitude for 90° rect excitation pulse
+
+    # User params (defined in the original JEMRIS XML file)
+    FOVx = 230e-3           # 230mm Field-of-View in x
+    Nx = 100                # Number of points in x
+    FOVy = 230e-3           # 230mm Field-of-View in y
+    Ny = 100                # Number of points in y
+    TE = 100e-3             # 100ms Time to Echo
+    Δtadc = 10e-6           # 10us ADC sampling time
+    Smax = 10/γ_jemris      # Gradient slew rate in (T/m/s)
```

Comment:
```suggestion
    Smax_jemris = 10                                    # Specify units [XX/XX]
    Smax = Smax_jemris/γ_rad_us_mT      # Gradient slew rate in (T/m/s)
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/c6ec91b5c89bc94b578a6af803e90f3709dcb5cb